torchci/clickhouse_queries/oss_ci_benchmark_llms/query.sql

```diff
@@ -6,13 +6,13 @@ WITH benchmarks AS (
         replaceOne(o.head_branch, 'refs/heads/', '') AS head_branch,
         o.workflow_id AS workflow_id,
         o.job_id AS job_id,
-        o.model.name AS model,
-        o.model.backend AS backend,
-        o.model.origins AS origins,
-        o.metric.name AS metric,
-        floor(arrayAvg(o.metric.benchmark_values), 2) AS actual,
-        floor(toFloat64(o.metric.target_value), 2) AS target,
-        o.benchmark.dtype AS dtype,
+        o.model.'name' AS model,
```

Comment:
One of of confusing aspect of ClickHouse is that sometimes `model.name` works, sometimes it needs to be `model.'name'`, and the other times it is `tupleElement(model, 'name')`.  The last one is probably the clearest because the field is a tuple, but it's also the lengthiest